easybuild/easyconfigs/m/Minimac/Minimac-2013.7.17-goolf-1.4.10.eb

```diff
@@ -0,0 +1,45 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
+
+name = 'Minimac'
+version = '2013.7.17'
+versionsuffix = '-beta'
```

Comment:
on second thought, just specifying a datestamp as 'version' as using that in the source name should do it, no need to add a hash as a version suffix (the datestamp serves that purpose, together with the specified MD5sum being checked)